leapfrog.rst

```diff
@@ -92,11 +92,11 @@ containers on CONTAINERS_TO_DESTROY as this will override the default that inclu
 
     export CONTAINERS_TO_DESTROY='all_containers:!galera_all:!neutron_agent:!ceph_all:!rsyslog_all:!elasticsearch_all:!swift_all'
 
-**Note:** *Currently the rpc-upgrades repo targets r14.21.0.  If you want to deploy the previous version you can:*
+**Note:** *Currently the rpc-upgrades repo targets r14.23.0.  If you want to deploy the previous version you can:*
 
 .. code-block:: shell
 
-   export RPC_TARGET_CHECKOUT=r14.20.0
+   export RPC_TARGET_CHECKOUT=r14.22.0
```

Comment:
That should be r14.23.0

Reply:
I should probably remove that part... it's part of the docs that gives an example if you want to deploy a release other than master so it shows an older version in the docs.  It's confusioning and it needs to change each time, so i'll remove that section in another commit so we don't have to reset jobs.

Reply:
Yes thats fine